documentation/API-design-guidelines.md

```diff
@@ -537,78 +539,125 @@ These considerations are below:
 
 ## 5. Versioning
 
-### 5.1 Versioning Strategy
+Versioning is a practice by which, when a change occurs in the API, a new version of that API is created.
 
-Service versioning is a practice by which, when a change occurs in the API of a service, a new version of that service is released so that the new version and the previous one coexists for a certain period of time.
+API versions use a numbering scheme in the format: x.y.z
 
-Consumers will be migrated to the new version of the service sequentially. When everyone is consuming the latest version of the service, the previous version is removed.
+* x, y and z are numbers corresponding to MAJOR, MINOR and PATCH versions.
+* MAJOR, MINOR and PATCH refer to the types of changes made to an API through its evolution.
+* Depending on the change type, the corresponding number is incremented.
+* This is defined in the [Semantic Versioning 2.0.0 (semver.org)](https://semver.org/) standard.
 
-Consumers can distinguish between one version of the service and another, the technique of adding the API version to the context of the base URL will be used, since this technique is the most used in the main reference APIs.
+### 5.1 API version (OAS Info object)
 
-The structure of the URL would have the following form:
-```http
-https://host:port/api/v1/resource
+The API version is defined in the "version" field (in the Info object) of the OAS definition file of an API. 
+
+```yaml
+info:
+  title: Number Verification
+  description: text describing the API
+  version: 2.2.0  
+  ...
+```
+
+In line with Semantic Versioning 2.0.0, the API with MAJOR.MINOR.PATCH version number, increments as follows:
+
+1. The MAJOR version when an incompatible / breaking API change is introduced
+2. The MINOR version when functionality is added that is backwards compatible
+3. The PATCH version when backward compatible bugs are fixed
+
+For more details on MAJOR, MINOR and PATCH versions, and how to evolve API versions, please see [API versioning](https://wiki.camaraproject.org/x/a4BaAQ) in the CAMARA wiki. 
+
+It is recommended to avoid breaking backwards compatibility unless strictly necessary: new versions should be backwards compatible with previous versions. More information on how to avoid breaking changes can be found below.
+
+### 5.2 API version in URL (OAS servers object)
+
+The OAS file also defines the API version used in the URL of the API endpoint (in the servers object).
+
+The API version in the URL only includes the "x" (MAJOR version) number of the API version as follows:
+
+```yaml
+servers:
+    url: {apiRoot}/qod/v2
 ```
 
+NOTE: CAMARA exceptionally allows to use both the MAJOR and the MINOR version number (v0.y) separated by a dot (".") in initial API versions (x=0) in the API version in the URL. 
```

Comment:
The maturity states initial & stable are described only later in the doc L598. Would be better to have this info before it being used here.

IMHO,  the statement "CAMARA exceptionally allows to use both the MAJOR and the MINOR version number" does not clearly state that this is a "requirement" by Camara.

Would prefer a simpler format --- Camara APIs with initial maturity state **shall** use both the MAJOR and the MINOR version number (v0.y) separated by a dot (".") in the API version ......

Reply:
Was it agreed that this is a MUST ?

Reply:
@tanjadegroot we had the discussion, but might have taken no final decision. But I support the clear line to make it mandatory - using v0 without the minor version would only work if there is only one initial version (which is a corner case).